api/config.py

```diff
@@ -212,6 +223,8 @@ def initialize_db():
         upsert=True
     )
 
+    log_db.access_log.create_index('origin.id')
```

Comment:
Thoughts on creating an index on the timestamp as well? Otherwise most log-grep operations will be very inefficient.

I had a look through any concerns about using mongo as a time series, and mostly TLDR'd out, but there didn't seem to be anything big. Most common facet is that folks suggest combining multiple events into one document, say one per minute, and so reduce per-second logging by a factor of 60. 

:face_with_rolling_eyes: <-- emoji Github declines to render because they are fascists

So yeah. Index?